components/homme/src/theta-l_kokkos/cxx/CaarFunctorImpl.hpp

```diff
@@ -334,9 +334,12 @@ struct CaarFunctorImpl {
     profiling_resume();
 
     GPTLstart("caar compute");
-    Kokkos::parallel_for("caar loop pre-boundary exchange", m_policy_pre, *this);
+    int nerr;
```

Comment:
Despite being ok, I wonder if valgrind will complain with something like "conditional jump may depend on uninitialized value" down below.

Reply:
Ah right, I forgot about ||reduce doing the init. Nvm then.